src/backend/drive/settings.py

```diff
@@ -737,6 +739,7 @@ class Development(Base):
     CORS_ALLOW_ALL_ORIGINS = True
     CSRF_TRUSTED_ORIGINS = ["http://localhost:8072", "http://localhost:3000"]
     DEBUG = True
+    LOAD_DEMO_URLS = True
```

Comment:
This settings is useless IMO, you should load the urls only if `DEBUG` is True.

Reply:
Not sure to understand. We want to add the urls only if DEBUG is True. So there is no need to change this settings from the tests.

Reply:
-> We discuss during a call about this, we decided to keep `LOAD_DEMO_URLS` because `DEBUG` loads debug toolbars that make the tests fail.